roles/tests-integration/lib/sniffer.rs

```diff
@@ -80,14 +75,12 @@ impl InterceptMessage {
         expected_message_type: MsgType,
         replace_message: PoolMessages<'static>,
         replace_message_type: MsgType,
-        break_on: bool,
```

Comment:
I dont mind removing this tbh. It was added because it can be useful to decide to stop the connection between two roles when a specific message is exchanged. Some of those cases are now covered by the `check_on_drop` flag thus I think it is cool to remove it.